.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -81,6 +81,7 @@ on:
           - '["Linux", "gen12"]'
           - '["amdgpu"]'
           - '["Linux", "arc"]'
+          - '["debug"]'
```

Comment:
I can rename the label. How about `cts-cpu`?

Reply:
Sure, that would be more meaningful.